src/index.html

```diff
@@ -3,11 +3,60 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars--0 stars">
```

Comment:
```suggestion
    <div class="stars stars--0">
```
Хоч і працює так само, але краще писати модифікатор ПІСЛЯ блоку